docker-compose-installed.yaml

```diff
@@ -122,6 +94,24 @@ services:
         tail -f /dev/null \
         \""
 
+# ---== Intkey and XO TPs have been moved to a separate repo ==---
```

Comment:
The wording of this comment refers to the changes in this PR. In a few months, "have been moved" will be old news. Instead, I suggest saying "are in a separate repo".